docs/API/NuGet-Client-SDK.md

```diff
@@ -0,0 +1,32 @@
+---
+# required metadata 
+
+title: NuGet v3 Client and NuGetGallery APIs | Microsoft Docs
+author: kraigb
+ms.author: kraigb
+manager: ghogen
+ms.date: 1/9/2017
```

Comment:
Update this to be a current date, e.g. 10/30/2017. This is the date that's shown on the live page and communicates freshness.

Reply:
This is fixed.